salt/utils/etcd_util.py

```diff
@@ -202,25 +404,32 @@ def get(self, key, recurse=False):
 
         return self.tree(key)
 
-    def read(self, key, recursive=False, wait=False, timeout=None, waitIndex=None):
-        """
-        Read a value of a key.
+    def read(
+        self, key, recurse=False, wait=False, timeout=None, waitIndex=None, **kwargs
+    ):
+        recursive = kwargs.get("recursive", None)
+        if recursive is not None:
+            salt.utils.versions.warn_until(
+                "Argon",
+                "Starting with the Argon release, the recursive kwarg will be deprecated."
```

Comment:
**suggestion** s/deprecated/removed. It's a terminology thing that we've struggled with for a long time :sweat_smile: Generally it should read something to the effect:

"The recursive argument has been deprecated, and will be removed in the Argon release. Use `recurse` instead."

We also need a `.deprecated` changelog entry (60325.deprecated) listing everything that we're deprecating from here that will be removed in the Argon release :)